packages/ui-components/src/textarea/textarea.tsx

```diff
@@ -126,7 +129,12 @@ function TextArea({
         if (keydownFilter && !keydownFilter(e)) {
             e.preventDefault();
         }
+        if (e.key === Key.ENTER && e.shiftKey && onComplete) {
+            return;
+        }
         if (e.key === Key.ENTER && onComplete) {
+            e.preventDefault(); // stops the enter being entered as a character to the textarea component
+            setTextAreaValue(''); // clear the text area on complete
```

Comment:
Shouldn't clearing the text be the responsibility of the parent in controlled mode? I'm not sure it belongs here, seems quite specific to the chat component

Reply:
Yes, forgot to remove this, done now